src/assets/scripts/client/aircraft/AircraftModel.js

```diff
@@ -2753,4 +2753,30 @@ export default class AircraftModel {
             this.setIsFlightStripRemovable();
         }
     }
+
+    /**
+     * Ensure that the provided altitude is valid
+     *
+     * @for AircraftModel
+     * @method validateNextAltitude
+     * @param nextAltitude {number} altitude the aircraft should maintain
+     * @return {array|undefined}           [success of operation, readback]
+     */
+    validateNextAltitude(nextAltitude) {
```

Comment:
I like where your head is head here, however, this introduces some un-necessary complexity because of the variadic return.  at the very least, the return of this method should be consistent.

Reply:
done